features/storage/security.feature

```diff
@@ -93,10 +101,10 @@ Feature: storage security check
     And the output should contain "Hello OpenShift Storage"
 
     Examples:
-      | storage_type         | volume_name | type   |
-      | gcePersistentDisk    | pdName      | gce    | # @case_id OCP-9700
-      | awsElasticBlockStore | volumeID    | ebs    | # @case_id OCP-9699
-      | cinder               | volumeID    | cinder | # @case_id OCP-9721
+      | storage_type         |
+      | gcePersistentDisk    | # @case_id OCP-9700
+      | awsElasticBlockStore | # @case_id OCP-9699
+      | cinder               | # @case_id OCP-9721
```

Comment:
Updating parameters will require update in Polarion, thus break 3.11 automation for those test cases.

Reply:
Yes, once the PR is merged, I'll update polarion.
This branch is for OCP 4.x only, right?

Reply:
updated the PR to keep the parameters.